models/video.model.js

```diff
@@ -0,0 +1,42 @@
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+const videoSchema = new Schema({
+  id: String,
+  snippet: {
+    publishedAt: String,
+    title: String,
+    description: String,
+    thumbnails: {
+      default: {
+        url: String,
+        width: Number,
+        height: Number
+      },
+      medium: {
+        url: String,
+        width: Number,
+        height: Number
+      },
+      high: {
+        url: String,
+        width: Number,
+        height: Number
+      },
+      standard: {
+        url: String,
+        width: Number,
+        height: Number
+      },
+    },
+    channelTitle: String,
+    statistics: {
+      viewCount: Number,
+      likeCount: Number,
```

Comment:
How are you updating the likes if its just a no? 
Shouldnt it be user ids??

Reply:
This model is used to render videos on frontend, no. of likes are stored in the user model.